db/model/ExplorerViews.ts

```diff
@@ -195,7 +195,7 @@ async function iterateExplorerViews(
             await explorer.updateGrapherFromExplorer()
 
             // Extract the generated config from the explorer's grapher state
-            const config = explorer.grapherState.toObject(false)
+            const config = explorer.grapherState.toObject()
```

Comment:
Probably debatable but I don't think we need to write the defaults into the database since we don't do this for any other config?